client/src/routes/SingleAssetPage/SingleAssetPage.js

```diff
@@ -178,6 +254,32 @@ const SingleAssetPage = () => {
           )}
         </section>
       </Container>
+
+      <Modal
+          backdrop="static"
+          show={exportAsset}
+          onHide={handleExportAssetFalse}
+        >
+        <Modal.Header closeButton>
+          <Modal.Title>Export {asset.name} To CSV</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <Row className="m-3">
+            <ConditionalAlert type={alertType3} message={alertMessage3} />
+          </Row>
+          <label>CSV File Name:</label>
+          <input type="text" value={csvName} onChange={(e) => setCsvName(e.target.value)} />
```

Comment:
Maybe just get the csvName without ".csv" then add csv right before saving. That way we can make sure users are only downloading csv files:
<img width="571" alt="Screenshot 2023-05-01 at 2 54 54 PM" src="https://user-images.githubusercontent.com/91560030/235520391-7869e17b-251d-45c0-8756-e7d8544f7f74.png">

<img width="627" alt="Screenshot 2023-05-01 at 2 55 22 PM" src="https://user-images.githubusercontent.com/91560030/235520375-18ad6c25-7253-4eda-8a35-8c050745f8df.png">